tools/codegen/dest/__init__.py

```diff
@@ -1,3 +1,6 @@
-from .register_dispatch_key import RegisterDispatchKey as RegisterDispatchKey
-from .register_dispatch_key import gen_registration_helpers as gen_registration_helpers
+from .register_dispatch_key import (
+    RegisterDispatchKey as RegisterDispatchKey,
+    gen_registration_helpers as gen_registration_helpers,
+    gen_registration_headers as gen_registration_headers,
```

Comment:
Why the `as` here?

Reply:
`mypy` needs it for some reason. It doesn't make sense to me either.